test/unit/test_command.rb

```diff
@@ -33,60 +33,65 @@ def test_leading_and_trailing_space_is_stripped
     def test_including_the_env
       SSHKit.config = nil
       c = Command.new(:rails, 'server', env: {rails_env: :production})
-      assert_equal %{( export RAILS_ENV="production" ; /usr/bin/env rails server )}, c.to_command
+      assert_equal %{( export RAILS_ENV=\"production\" ; /usr/bin/env rails server )}, c.to_command
```

Comment:
This seems incorrect to me. The backslashes mean that the quotes will be included in the value. For example if I run this locally:

```
$ export RAILS_ENV=\"production\" ; echo $RAILS_ENV
"production"
```

Reply:
this is just a display problem, it is the correct string:

```
puts %{( export RAILS_ENV=\"production\" ; /usr/bin/env rails server )}
( export RAILS_ENV="production" ; /usr/bin/env rails server )
```

I removed the extra escapes from the assertion and it looks saner now